### PR TITLE
Refactored YAML test input

### PR DESCRIPTION
**Pull request by flbulgarelli**
Fixes https://github.com/mumuki/mumuki-platform/issues/303

### CODE REVIEW

**Comment by laendoor** [comment]
Hi @flbulgarelli 

I think I finished implementing `final_dataset` and all the cases and refactor that we discuss in #6.

Do you like to implement issues #3, #4 & #5 before make pull request?

Do you need me to take care of someone?

**Comment by flbulgarelli** [comment]
> I think I finished implementing final_dataset.

Yes, I was just looking at it! Awesome. 

> Do you like to implement issues #3 #4 & #5 before make pull request?

No, I was actually going to review this PR and merge it as-is. I'd suggest going on #3 in another branch.

#4 and #5 are actually issues of  https://github.com/mumuki/mumuki-styles.

**Comment by flbulgarelli** [comment]
Also, any help or suggestions with those issues are welcome, particularly with issue #3.  I will start working on it after Tuesday probably

File: lib/version_hook.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module SqliteVersionHook
-  VERSION = '2.2.1'
```
Comment (flbulgarelli) [inline lib/version_hook.rb:2]: I'd recommend to do those kind of commits and tags directly from master branch. It is not a big deal, just for future releases.

Comment (laendoor) [reply lib/version_hook.rb:2]: It's true, I did not realize.

**Pull request merged**